src/index.tsx

```diff
@@ -1,15 +1,40 @@
 import './wdyr'
 import 'lib/polyfills'
 
+import * as Sentry from '@sentry/react'
 import { App } from 'App'
 import { AppProviders } from 'AppProviders'
+import { getConfig } from 'config'
 import React from 'react'
 import { createRoot } from 'react-dom/client'
 import { renderConsoleArt } from 'lib/consoleArt'
 import { reportWebVitals } from 'lib/reportWebVitals'
 
 import * as serviceWorkerRegistration from './serviceWorkerRegistration'
 
+Sentry.init({
+  ...(window.location.hostname === 'localhost'
+    ? {}
+    : { dsn: getConfig().REACT_APP_SENTRY_DSN_URL }),
+  attachStacktrace: true,
+  denyUrls: ['alchemy.com'],
+  integrations: [
+    // Sentry.browserTracingIntegration(),
+    // Sentry.replayIntegration(),
```

Comment:
Allows session replay around the issue.
Potentially a nice addition, and Sentry lets us hide text inputs as well as all text (see https://docs.sentry.io/platforms/javascript/session-replay/privacy/) which would be the main issue re: exposing critical data like seeds or rugging users privacy re: addresses.

However if we want to go with this, we'll want to triple check that everything critical *is* indeed hidden with no exception whatsoever.